docs/monitoring/checkpoint_monitoring.md

```diff
@@ -61,7 +61,7 @@ The checkpoint history keeps statistics about recently triggered checkpoints, in
 - **Trigger Time**: The time when the checkpoint was triggered at the JobManager.
 - **Latest Acknowledgement**: The time when the latest acknowledged for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
```

Comment:
Could you also fix that one while reiterating over the file, please:
```suggestion
- **Latest Acknowledgement**: The time when the latest acknowledgement for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
```

Reply:
Thanks for the review. Updated per suggested.